content/200-orm/800-more/300-upgrade-guides/200-upgrading-versions/index.mdx

```diff
@@ -26,15 +26,15 @@ To upgrade to the latest version of Prisma:
 
 ## Common upgrade paths
 
-### Prisma 2 and onwards
+### Prisma ORM 2 and onwards
 
-- [Upgrade to Prisma 5](/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-5)
-- [Upgrade to Prisma 4](/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-4)
-- [Upgrade to Prisma 3](/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-3)
+- [Upgrade to Prisma ORM 5](/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-5)
+- [Upgrade to Prisma ORM 4](/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-4)
+- [Upgrade to Prisma ORM 3](/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-3)
```

Comment:
For completelness we probably also should update these file names, but let's do that in a follow up PR, or reviewing this will be even harder (and doing the rename now might also cause us to loose some of the reviews and suggestions that exist).